custom/icds/tests/test_views.py

```diff
@@ -4,12 +4,34 @@
 
 
 class TestViews(TestCase):
```

Comment:
I'm having a very strange issue where if I run these tests individually they all pass, but if I run into more than one I get the `STATICI18N_OUTPUT_DIR` issue that @orangejenny mentions here: https://github.com/dimagi/commcare-hq/pull/21757

Hoping it's something weird with my config and not an issue on Travis but let's see... (they still failed locally even before I made any changes)